src/core_ocean/shared/mpas_ocn_vmix.F

```diff
@@ -821,9 +821,10 @@ subroutine ocn_vel_vmix_tend_implicit_spatially_variable_mannings(meshPool, forc
          end do
 
          ! average cell-based implicit bottom drag to edges and convert Mannings n to Cd
-        if (config_use_implicit_bottom_roughness) then
-           varmin=min(0.16_RKIND/(log(0.5_RKIND*0.5_RKIND*(bottomDepth(cell1)+bottomDepth(cell2))/0.001))**2.0,0.1)
-           implicitCd =max(0.0025_RKIND,varmin)
+         if (config_use_implicit_bottom_roughness) then
+           implicitCd = max(0.0025_RKIND, &
+                        min(0.1_RKIND, &
+                          0.16_RKIND/log(250.0_RKIND*(bottomDepth(cell1)+bottomDepth(cell2)))**2 ))
```

Comment:
@nairita87 please check this. I consolidated these two lines, putting min and max together, and removing the varmin variable.

Also, 0.5*0.5/0.001 = 250, so I just pre-computed the math.

Reply:
@mark-petersen , Mark, I verified, passes the test nicely
![commit_c7583d8](https://user-images.githubusercontent.com/62908159/116118983-25c83c80-a67b-11eb-8433-002001593736.png)